models/api.models.js

```diff
@@ -29,9 +30,16 @@ exports.selectApiEndPoints = () => {
                 exampleResponse: fileContents[key].exampleResponse
             }
         })
-        console.log(apiEndpoints)
         return apiEndpoints
     })
 }
 
+exports.selectArticleById = (id) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = $1`, [id])
+    .then(({rows}) => {
```

Comment:
Great use of the parameterized query for the article_id to avoid SQL injection 👍